Improve language and fix menu entry name of rqt (#1749)
ros2/ros2_documentation: source/Tutorials/Topics/Understanding-ROS2-Topics.rst

    rqt_graph

> You can also open rqt_graph by opening rqt via ``rqt`` and selecting **Plugins** > **Introspection** > **Node Graph**.

Reviewer: We are trying to move away from just using ``rqt`` and to using ``ros2 run rqt rqt`` instead (this will work on all platforms).  Because of that, I don't think we should particularly make this change, and just leave this as:

```
You can also open rqt_graph by opening ``rqt`` and selecting **Plugins** > **Introspection** > **Node Graph**.
```

  /turtle1/color_sensor
  /turtle1/pose

> ``ros2 topic list -t`` will return the same list of topics but this time with the topic type appended in brackets:

Reviewer: ```suggestion
``ros2 topic list -t`` will return the same list of topics, this time with the topic type appended in brackets:
```

  /turtle1/color_sensor [turtlesim/msg/Color]
  /turtle1/pose [turtlesim/msg/Pose]

> Thus topics are defined by names and types.

Reviewer: Honestly, I'd just delete this line.  It is redundant with the leading sentence above.

Nodes send data over topics using messages.
Publishers and subscribers must send and receive the same type of message to communicate.

> The topic types we saw earlier after running ``ros2 topic list -t`` let us know what type of messages is used on each topic.

Reviewer: ```suggestion
The topic types we saw earlier after running ``ros2 topic list -t`` let us know what message type is used on each topic.
```

This means that in the package ``geometry_msgs`` there is a ``msg`` called ``Twist``.

> Now we can run ``ros2 interface show <msg type>`` on a type to learn its details, specifically, what structure of data the message expects.

Reviewer: ```suggestion
Now we can run ``ros2 interface show <msg type>`` on this type to learn its details, specifically, what structure of data the message expects.
```

.. image:: rqt_graph3.png

> In this case, ``/turtlesim`` also publishes in the ``pose`` topic to which a new ``echo`` node is subscribed.

Reviewer: ```suggestion
You can see that the ``/turtlesim`` node is also publishing to the ``pose`` topic, which the new ``echo`` node is subscribed to.
```
